dask_cuda/local_cuda_cluster.py

```diff
@@ -177,6 +178,16 @@ def __init__(
                     "is not available. For installation instructions, please "
                     "see https://github.com/rapidsai/rmm"
                 )  # pragma: no cover
+            self.rmm_pool_size = parse_bytes(self.rmm_pool_size)
+        else:
+            if enable_nvlink:
+                warnings.warn(
+                    "When using NVLink we recommend setting a "
+                    "`rmm_pool_size` or setting an RMM pool via `client.run`. "
```

Comment:
Should we leave it at setting `rmm_pool_size`? The issue with `client.run` is if some workers are restarted after that call, they won't have a pool any more. Whereas `rmm_pool_size` always works since it is a worker plugin and will be rerun on worker startup.

Reply:
That's a good point.  While the can used client.run we should recommend using rmm_pool_size

Reply:
Thanks Ben! 😄